OWNERS_ALIASES

```diff
@@ -0,0 +1,23 @@
+aliases:
+  sig-release-approvers:
```

Comment:
how do you currently keep this in sync across repos? :)

Reply:
@dims -- I do a quarterly review/update on the SIG Release OWNERS, which is coming up soon.